src/modules/user-settings/components/editor-picker.tsx

```diff
@@ -39,6 +75,12 @@ export const EditorPicker = ( { value, onChange, disabled }: EditorPickerProps )
 							{ editorConfig.label }
 						</option>
 					) ) }
+
+					{ installedEditors.length === 0 && (
+						<option value={ '' } disabled>
+							{ __( 'None' ) }
+						</option>
+					) }
```

Comment:
I have added this `None` option when no editors are installed to clearly differentiate between the `Available` and `Not Installed` groups. 

![CleanShot 2025-05-08 at 12 33 14@2x](https://github.com/user-attachments/assets/6feb32e1-8cab-458a-bc5a-3fefc832c29d)